_conferences/ro-mobos-2020.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "RO MobOS"
```

Comment:
The "RO" prefix doesn't appear to be part of the conference name

Reply:
I've removed it.